sklearn/preprocessing/data.py

```diff
@@ -111,10 +120,14 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
 
     To avoid memory copy the caller should pass a CSC matrix.
 
+    See examples/preprocessing/plot_all_scaling.py for a comparison of the
```

Comment:
I was checking the API doc and it seems that this is a common way to do (plain text in Notes section).

Speaking with @lesteve, he was pointing out that references can be annoying when printing the docstring which is not the case with plain text.

We need to change it across the entire doc if we decide to make such changes. IMO, moving the examples section (the one on the bottom of each page) before the summary of the methods would be appropriate. However, it seems that you already tried and that it is tricky.